docs/Development/General/Refactoring_3.md

```diff
@@ -4,14 +4,19 @@ currentMenu: dev-general
 # Refactoring for 3.0
 
 ## Modules
+### File Structure
+The file structure of Zikula 3.0 is much different than the structure in 2.0. Intead of extensions being kept
+in /modules or /themes, they are placed into /src/extensions. Much of the code that handles rendering pages
+is outside the public folder that the webserver can see. To serve your html pages, you will need to navigate
+to zikula_folder/src/ to serve pages. A virtual host or modrewrite may be needed on your production server. 
 
```

Comment:
the bit from "to serve your html pages..." is not related to refactoring. it is about installation and this is covered in those docs.

Reply:
I was suggesting to remove from "to serve your html pages" to the end of the graph